src/colors.js

```diff
@@ -166,6 +166,35 @@ const primary = {
     800: "#225923",
     900: "#1d4a1f",
   },
+  grayBlue: {
+    25: "#fcfcfd",
+    50: "#f8f9fc",
+    100: "#eaecf5",
+    200: "#d5d9eb",
+    300: "#b3b8db",
+    400: "#717bbc",
+    500: "#4e5ba6",
+    600: "#3e4784",
+    700: "#363f72",
+    800: "#293056",
+    850: "#1B243F",
+    875: "#10162B",
+    900: "#0f0d2c",
+    950: "#080f25",
```

Comment:
I think we need to add 975 here. as it was added in https://github.com/krystal/katapult/commit/399da1a0afe095917b93ec7787f1e95cb32c42aa#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2e @philgwynne are you able to confirm this is the final list? Or should we hold-off merging for now?

Reply:
Yes, there's another that needs editing too. I'll add a suggestion.